client/pokemon/pokemon.ctrl.js

```diff
@@ -31,13 +31,7 @@ module.exports = class Pokemon {
       this.errorStatusCode = err.statusCode;
     }).then(() => this.$scope.$apply());
   }
-  parseBoxViewProps () {
-    this.parsedOt = replace3dsUnicodeChars(this.data.ot);
-    const tidSize = this.data.gen === 7 ? 6 : 5;
-    this.paddedTid = (this.data.idNo || this.data.tid).toString().padStart(tidSize, '0');
-    this.paddedEsv = this.data.esv.toString().padStart(4, '0');
-    this.parsedNickname = replace3dsUnicodeChars(this.data.nickname);
-
+  parseCardViewProps () {
```

Comment:
See line 46. He's not renaming parseBoxViewProps. He's just separating a portion of it so it can be called elsewhere (client/pokemon/pokemon-icon.view.html).